ActivatePubKey/handler.ts

```diff
@@ -60,11 +62,14 @@ export const activatePubKeyForAssertionRef = (
       pubKey,
       status: PubKeyStatusEnum.VALID
     }),
-    TE.mapLeft(error => {
-      const err = error.detail;
-      context.log.error(err);
-      return ResponseErrorInternal(err);
-    })
+    eventLog.taskEither.errorLeft(error => [
+      `${FN_LOG_NAME} | Error while writing pop document: ${error.kind} - ${error.detail} - ${error.message}`,
```

Comment:
We don't need to write the `FN_LOG_NAME` here, it's already defined in log trace `operation_Name` column.